libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -155,6 +155,16 @@ bool AsmAnalyzer::operator()(FunctionalInstruction const& _instr)
 	return success;
 }
 
+bool AsmAnalyzer::operator()(assembly::ExpressionStatement const& _statement)
+{
+//	size_t initialStackHeight = m_stackHeight;
+	bool success = boost::apply_visitor(*this, _statement.expression);
+//	if (!expectDeposit(0, initialStackHeight, _statement.location))
```

Comment:
Is this stray code needed?

Reply:
It will be re-enabled in https://github.com/ethereum/solidity/pull/3317/files#diff-d65e7a5103899854f9b92288d63d4859R162